### DISCUSSION TITLE
How do you close an original tab after splitting, and not close the split?

### DISCUSSION THREAD

**Discussion by octoxan** [LazyVim/LazyVim#932] [Q&A] [answered]
Trying to wrap my head around closing tabs.

I have 5 tabs open, and I split one to the right by doing leader + | and that splits my current tab to the right.

Now, how do I close the instance of it on the left so that there's 4 on the left and the 1 on the right?

If I go to window mode and switch back to the left tab, :q closes my new tab and then it's no longer split. If I do :tabclose it tells me "E784: Cannot close last tab page." even though I have 5 tabs open.

This is on macOS with a fresh nvim and fresh LazyVim install.

Is there some other way I'm supposed to split tabs and be able to close the original version of the tab on the left?

Thanks!

**Comment by dpetka2001** [comment, marked as answer]
These are not tabs. They are buffers. What you're doing is splitting the file on your current buffer to a new vertical window. But both windows are still on the same tab. You could switch the buffer on the left window to a different buffer and keep only the buffer that you split on the right window.

**Reply by octoxan** [reply]
How can you accomplish this, if you don't mind? I tried :bd as well once the left window is active and it still closes the new buffer and not the old buffer.

That or if there was a way to split tabs without ending up with multiples of the same tab everywhere, like maybe to just move a file to a new buffer on the right instead of copying it?

**Reply by dpetka2001** [reply]
When you `:bd` you unload the buffer from the memory and from the buffer list, thus it is not visible any more.Then you're left with whatever other buffer remains still open in your split windows. When you are on the left window just do `<leader>,` to show you the buffer list and choose the file that you want to be visible on the left window. Then you will have one file on the left window and another one on the right (the one you have already split before).

**Reply by octoxan** [reply]
Even using the mouse to click the "X" on the buffer on the left closes both of them, and not just the one for the X that was clicked. That's normal? Just making sure lol

**Reply by dpetka2001** [reply]
Read my previous response about `:bdelete`.

**Reply by folke** [reply]
Best to do some reading on Vim buffers/windows/tabs.

As long as you don't fully grasp what they are, you won't be able to properly use Neovim.

**Reply by octoxan** [reply]
Ah so you just can't have it function like a traditional editor with multiple tabs on the left and multiple tabs on the right?

**Reply by folke** [reply]
Maybe you can with plugins, who knows.
But my comment still applies. You really should understand what buffers/windows/tabs are first.

**Reply by dpetka2001** [reply]
You could set up bufferline to behave like a tabline explicitly, but you should first understand about how buffers/tabs behave on Neovim. If you read bufferline's docs I'm sure you'll find some examples.

**Reply by octoxan** [reply]
Alright I think I got it from this image I found. So each one is just a viewer of a certain file and not like traditional editors. I think I like this Vim way more now that I'm wrapping my head around it. You don't really have duplicate tabs open so there's no need to close the others. Got it. Thanks

![image](https://github.com/LazyVim/LazyVim/assets/4382195/da34c396-1591-4538-811e-fb78683b4aca)

**Reply by directormac** [reply]
If you want to sperate bufferlist for different tabs, use scope.nvim the resoring of session does not work well but it works well in seperating buffer list.

**Reply by dpetka2001** [reply]
@directormac Thank you for your suggestion. I'm trying it and looks nice so far.
